.circleci/config.yml

```diff
@@ -107,6 +107,11 @@ jobs:
               yarn build
               cd ../..
 
+              # Build move-granules task
```

Comment:
why do we need to build this task separately?

Reply:
Because this task isn't posted to npm yet and won't be until this is granule-storage stuff is merged to master